src/core/InternalAnimatedValue.js

```diff
@@ -32,6 +33,10 @@ export default class InternalAnimatedValue extends AnimatedNode {
   }
 
   constructor(value, constant = false) {
+    invariant(
+      value !== null,
+      'you are trying to set null on animated value'
```

Comment:
I think something like: "Animated.Value cannot be set to the null" would be more clearer as an error message